web-js/src/components/TodoItemsPage.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import {
+  Container,
+  Button,
+  Typography,
+  List,
+  LinearProgress,
+} from "@material-ui/core";
+import AddIcon from "@material-ui/icons/Add";
+import { TodoItem } from './TodoItem'
+import { DraftTodoItem } from './DraftTodoItem'
+import { useTodos } from "./useTodos";
+import { useDraftTodos } from "./useDraftTodos";
+import { useShowLoader } from "./util-hooks";
+import { MoreInfoTemplateAndDocs, MoreInfoGraphiQL } from './MoreInfo'
+
+export function TodoItemsPage() {
+  const { loading, todos, ...todoActions } = useTodos();
+  const { draftTodos, ...draftTodoActions } = useDraftTodos();
+  const showLoader = useShowLoader(loading, 200);
```

Comment:
I'm a bit confused by this loader. Why the artificial 200ms loader?

Reply:
The 200ms here means that the loader should only show if the request takes longer than 200ms to complete. It's a similar idea to React Suspense - it's jarring and unnecessary to see loaders for things that load fast, so you should wait a little bit to render them.